src/Toolkit/Str.php

```diff
@@ -1018,6 +1018,18 @@ public static function snake(string $value = null, string $delimiter = '_'): str
         return $value;
     }
 
+    /**
+     * Convert a string to studly caps case.
+     *
+     * @param string $value The string to convert.
+     * 
+     * @return string
+     */
+    public static function studly(string $value): string
```

Comment:
For consistency with the other case-altering `Str` methods, the method signature should be:

```suggestion
    public static function studly(string $value = null): string
```

We do this for Postel's law so that an accidental `null` won't cause a PHP error and will be treated like an empty string. So for `null` the return value should be an empty string.
However the documented signature should still only support a string, so the docblock and the `string` type are unchanged.